src/library/Uno.Material/Styles/Controls/NavigationView.xaml

```diff
@@ -493,6 +493,7 @@
 										Style="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=IsPaneOpen, Converter={StaticResource OpenToPaneButtonStyle}}"
 										AutomationProperties.LandmarkType="Navigation"
 										Visibility="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.PaneToggleButtonVisibility}"
+									    Background="Transparent"
```

Comment:
That's a nice simple solution. 
Also, it seems to partially fix #121. Though there is still a grayed background from the button pressed state.

Reply:
The buttons, style should be available in the NavigationView.xaml, a fix would be to possibly animate the opacity for a few seconds rather then just having a setter on the pressed visual state.

Reply:
Still the hitbox will be wrong:
normal: 
![image](https://user-images.githubusercontent.com/32882405/83800629-b669e880-a675-11ea-80a5-3dde73837d91.png)
light-dismissed: 
![image](https://user-images.githubusercontent.com/32882405/83800654-c7b2f500-a675-11ea-9f79-df8288e072b6.png)